CRVResponse/fcl/epilog_extracted_v03.fcl

```diff
@@ -2,6 +2,7 @@ physics.producers.CrvPhotons.crvStepModuleLabels  : ["CrvSteps"]
 physics.producers.CrvPhotons.crvStepProcessNames  : ["CrvStepsConverter"]
 physics.producers.CrvPhotons.CRVSectors           : ["EX","T1","T2"]
 physics.producers.CrvPhotons.reflectors           : [  0,   1,   0 ]
+physics.producers.CrvPhotons.scintillationYields  : [39400,39400,39400]
```

Comment:
I suppose we need to update the light yield with the latest measurements from Wideband

Reply:
> I suppose we need to update the light yield with the latest measurements from Wideband
Is it important to do this before we start the upcoming production?